app/controllers/socials_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# Controller that manages socials
```

Comment:
I would remove this comment. In general I'm not a fan of comments unless they really add something. Most times you can get rid of them with clearer naming. This one doesn't explain anything new, imo. What do you think?

Reply:
I didn't want it there but rubocop forced me :sweat_smile: